Shared/Samples/Animate images with image overlay/AnimateImagesWithImageOverlayView.swift

```diff
@@ -0,0 +1,177 @@
+// Copyright 2023 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import SwiftUI
+
+struct AnimateImagesWithImageOverlayView: View {
+    /// The view model for the sample.
+    @StateObject private var model = Model()
+    
+    /// The text for the start stop button.
+    @State var startStopButtonText = "Start"
+    
+    /// A Boolean value indicating whether the speed options sheet is showing.
+    @State private var isShowingSpeedOptions = false
+    
+    var body: some View {
+        VStack {
+            // Create a scene view to display the scene.
+            SceneView(scene: model.scene, imageOverlays: [model.imageOverlay])
+                .onAppear {
+                    // Create display link and load first image.
+                    model.displayLink = model.makeDisplayLink()
+                    model.setImageFrame()
+                }
+                .onDisappear {
+                    // Invalidate display link before exiting.
+                    model.displayLink.invalidate()
+                }
+            VStack {
+                HStack {
+                    Slider(value: $model.imageOverlay.opacity, in: 0.0...1.0, step: 0.01)
+                    VStack {
+                        Text("Opacity")
+                        Text(model.imageOverlay.opacity, format: .percent.precision(.fractionLength(0)))
+                    }
+                }
+                .padding([.top, .horizontal])
+                HStack {
+                    Button(startStopButtonText) {
+                        startStopButtonText = startStopButtonText == "Start" ? "Stop" : "Start"
+                        model.displayLink.isPaused.toggle()
+                    }
+                    Spacer()
+                    Menu("Speed") {
+                        Picker("", selection: $model.displayLink.preferredFramesPerSecond) {
```

Comment:
When I change the selection of the picker, the checkmark is not at the correct row.